tests/unit/cluster/cluster-shards.tcl

```diff
@@ -0,0 +1,54 @@
+# Get the node info with the specific node_id from the
+# given reference node. Valid type options are "node" and "shard"
+proc get_node_info_from_shard {id reference {type node}} {
+    set shards_response [R $reference CLUSTER SHARDS]
+    foreach shard_response $shards_response {
+        set nodes [dict get $shard_response nodes]
+        foreach node $nodes {
+            if {[dict get $node id] eq $id} {
+                if {$type eq "node"} {
+                    return $node
+                } elseif {$type eq "shard"} {
+                    return $shard_response
+                } else {
+                    return {}
+                }
+            }
+        }
+    }
+    # No shard found, return nothing
+    return {}
+}
+
+start_cluster 3 3 {tags {external:skip cluster}} {
+
+test "Cluster should start ok" {
```

Comment:
Let's indent this and all other blocks here in the new code (Madelyn said this in another thread).